### ISSUE TITLE
Webhooks UI accessibility assessment and fixes

### ISSUE THREAD

**Issue by dibbles** [tektoncd/experimental#63]
The webhooks extension user interface needs looking over for a11y issues and any fixes needed should be made.

**Comment by carlos-logro** [comment]
@dibbles After reviewing I see that all the text fields & dropdowns have their respective `labelText` although hidden (not sure if its hidden from screen readers). They also have a div that comes before each field that works as a label too. 

@AlanGreene do these `labelText` value work as aria-labels? Lastly, the only place where I did not see any a11y values were on the tooltips. Should they have one?

**Comment by AlanGreene** [comment]
`labelText` does act as an accessible name as it's used for the content of a `<label for="...">{labelText}</label>`

The accessibility tab in the Chrome dev tools is very useful to inspect this type of thing.

![image](https://user-images.githubusercontent.com/2829095/60132633-d3bee300-9793-11e9-9446-b67ef4c584a3.png)

Depending on how things are hidden visually, they may or may not be hidden from screen readers.

**Comment by carlos-logro** [comment]
@dibbles @AlanGreene @mnuttall 

<img width="780" alt="Screen Shot 2019-06-26 at 2 49 29 PM" src="https://user-images.githubusercontent.com/49996607/60207292-7f634400-9823-11e9-9505-bd281d9afa57.png">

<img width="714" alt="Screen Shot 2019-06-26 at 2 49 38 PM" src="https://user-images.githubusercontent.com/49996607/60207301-84c08e80-9823-11e9-9548-d9ca00998656.png">

<img width="723" alt="Screen Shot 2019-06-26 at 2 49 51 PM" src="https://user-images.githubusercontent.com/49996607/60207311-88541580-9823-11e9-860a-d384ad712ba1.png">

So after using the accessibility tool in chrome and discarding the invalid `aria attribute` in the side nav, we can say the page got a perfect score. Now, one of the passed audits (that does not makes sense to me) is the elements not having `tabIndex greater than 0` because they indeed have declared values greater than 0. But the tool does have a point when it comes to backing their reason behind this case: `"A value greater than 0 implies an explicit navigation ordering. Although technically valid, this often creates frustrating experiences for users who rely on assistive technologies."` With that said, I suggest we remove the tooltips & display does help texts in italics below the label just like it appears in the `Import Tekton resources` page. This not only follows the rule stated above but as well fix the tabIndex ordering from issue #66 just because they will go from one text field to another in a natural sequential order while also maintaining a consistent design throughout the whole dashboard. What do you guys think?

**Comment by AlanGreene** [comment]
I agree with the guidelines that non-zero tab index should be avoid where possible. Most of the time it's not required as the natural focus order provides the expected flow.

As for the tooltips, many of the form elements accept a `helperText` prop that will display this information below the label. This is the approach used on the Import page. See the form usage guidelines for more:
- https://www.carbondesignsystem.com/components/form/usage#labels
- https://www.carbondesignsystem.com/components/form/usage#help-text

**Comment by carlos-logro** [comment]
kept checking the a11y specs with the IBM DAP tool & everything seemed to be fine even with the new tooltips I added.

**Comment by a-roberts** [comment]
@carlos-logro @AlanGreene are we good to close this now as it was specific to issues found at the time? 

I'm inclined to add something to our DEVELOPMENT.md along the lines of running the IBM DAP tool now (at https://aat.w3ibm.mybluemix.net/tools/auth/tools/dap.html) which anyone can use, and advocating for its usage as part of our publishing process, and then for any new issues we find, we'll be creating new issues to track their implementation.

**Comment by AlanGreene** [comment]
👍 fine with closing it, any other problems found can be handled through new issues.
The DAP link you gave above is internal, so wouldn't be available to anyone outside IBM.

**Comment by a-roberts** [comment]
> 👍 fine with closing it, any other problems found can be handled through new issues.
> The DAP link you gave above is internal, so wouldn't be available to anyone outside IBM.

Yeah, on the hunt for a freely available one atm, just the one problem so far reported above so I'm gonna close this one, cheers 😄
